.github/workflows/pr_build.yml

```diff
@@ -10,6 +10,9 @@ jobs:
     env:
       TAMAGO_VERSION: 1.21.3
       TAMAGO: /usr/local/tamago-go/bin/go
+      LOG_ORIGIN: TEST-LOG
```

Comment:
Maybe an origin like: `transparency.dev/armored-witness/firmware_transparency/pr_ci/0`. All caps log origins, even for a throwaway log, gives me the shudders.

Also a comment making it clear around here that this is not the real CI log would be useful.

Reply:
sure, although I'll see if I can come up with an origin which avoids creating confusion about CI in the first place.